sdks/java/build-tools/src/main/resources/beam/checkstyle.xml

```diff
@@ -100,7 +100,11 @@ page at http://checkstyle.sourceforge.net/config.html -->
     </module>
 
     <module name="IllegalImport">
-      <property name="illegalPkgs" value="com.google.api.client.repackaged, com.google.api.client.util.util.ByteStreams, com.google.api.client.util.Charsets, com.google.api.client.util.Collections2, com.google.api.client.util.Joiner, com.google.api.client.util.Lists, com.google.api.client.util.Maps, com.google.api.client.util.Objects, com.google.api.client.util.Preconditions, com.google.api.client.util.Sets, com.google.api.client.util.Strings, com.google.api.client.util.Throwables, io.netty.util.internal"/>
+      <property name="illegalPkgs" value="com.google.api.client.repackaged, io.netty.util.internal"/>
+    </module>
+
+    <module name="RegexpSinglelineJava">
```

Comment:
Maybe a comment about why we're using Regexp rather than IllegalImport? My guess is that illegalPkgs doesn't work with classes.